app/views/question_flow/_benefit_section.html.slim

```diff
@@ -21,17 +20,19 @@
       p.govuk-error-message
         span.govuk-visually-hidden = t("generic.error_prefix")
         = model.errors.messages[:benefit_type].first
-    input.govuk-input.govuk-input--width-20 [
-      autocomplete="off"
-      data-suggestions="suggestions"
-      data-module="govuk-input"
-      type="text"
-      name="benefit_model[items][#{counter}][benefit_type]"
-      value=model.benefit_type
-      id=id_pattern.gsub("ID", counter.to_s)
-      data-add-another-dynamic-elements="id:#{id_pattern},name:benefit_model[items][ID][benefit_type]"
-      class=("govuk-input--error" if model.errors.messages[:benefit_type].any?)
-    ]
+    .govuk_text_field
```

Comment:
Think we need to add an aria-owns=1-type-suggestions here. JS sets an aria-owns on the input box parent which is redundant (apparently aria-owns is to communicate ownership when the relationship isn't specified in the DOM)

Reply:
Have added `aria-owns="suggestions"`, which I think does the job and also removed adding `aria-owns` from the JS which was adding it to the parent node.